### DISCUSSION TITLE
Using IBEX on regular slides vs 2-well chambered coverglass

### DISCUSSION THREAD

**Discussion by mickelj7** [IBEXImagingCommunity/ibex_imaging_knowledge_base#193] [General]
I'm starting to finalize my procedure to validate our IBEX process for my lab. I am going to try to complete this process as the Manuel method with regular slides rather then the 2-well chambered coverglass. If anyone has tried this method I would appreciate any input on major changes to the IBEX procedure that might occur due to the differences. 
I will be trying to use a PAP pen to still allow pooling of solutions, as this is the method I have used during IHC methods in the past. 
Thank you for any input or recommendations.

**Comment by radtkea** [comment]
Hi Jaide,

Thanks for asking your questions in the open! I alerted [Anita Gola](https://www.nature.com/articles/s41586-020-2977-2) about your mouse liver question. 

Yes, we use slides all the time! In fact, our original [PNAS](https://www.pnas.org/doi/full/10.1073/pnas.2018488117) paper describes the manual IBEX workflow for tissues sectioned onto slides. This figure (feel free to take/use) is Figure S3A from our PNAS paper. 
![Manual_IBEX_Workflow](https://github.com/user-attachments/assets/b761f3a3-9175-499d-a535-a0f36a372f2b).

Here's a basic workflow. 
1. Prepare [chrome alum-gelatin](https://www.newcomersupply.com/product/chrome-alum-gelatin-adhesive/) coated slides. Be mindful of the expiration date. I always put a calendar alert 3 months before the expiration date. Tissue adherence is compromised with expired adhesive. It also takes awhile to ship from WI to MD. If located overseas, then you can make your own adhesive using [Sonja's recipe](https://www.protocols.io/view/self-made-chrome-alum-gelatin-coated-slides-3byl49kkogo5/v1)! Follow this [video](https://www.youtube.com/watch?v=ksNR3gsl5rg) created by Ian on how to coat slides using the adhesive. 
2. Section tissue onto coated glass slides. 
3. Place in a 60°C oven for 1 hour to adhere the tissues to the slides. I also let the tissue dry overnight at room temperature and perform my immunolabeling in the morning.
4. Use a [pap pen](https://vectorlabs.com/products/immedge-hydrophobic-barrier-pen) to outline your tissue sample. 
![Pap_Pen](https://github.com/user-attachments/assets/d115d85a-d68f-462b-8a40-bc20c8d82cce)
5. Rehydrate your tissue with 1X PBS. Incubate for 5 minutes at RT.
6. Perform blocking and immunolabeling as you would for standard multiplexed imaging. 
7. Mount your sample with aqueous mounting media. This is very important because you need to wash off the mounting media for dye inactivation, immunolabeling, and imaging of the subsequent rounds. We always use [Fluoromount-G](https://www.southernbiotech.com/fluoromount-gr-0100-01). Here's a helpful [video](https://www.youtube.com/watch?v=tqHlrSmsG_8) showing how to mount your samples without bubbles prepared by Ian. 
8. Image your samples. It will vary depending on your microscope, of course, but the tips in [Box 2](https://www.nature.com/articles/s41596-021-00644-9) for "Microscope set up and imaging" for Manual IBEX are generally helpful.
9. After imaging, soak off your coverslip as demonstrated [here](https://www.youtube.com/watch?v=gXN19na8I6Y). 
10. Perform dye inactivation using LiBH4. Follow all institutional safety instructions. Please see [video](https://www.youtube.com/watch?v=-MH_aLEMyF8).

Please reach out if you have any more questions! Happy to chat with you (andrea.radtke "@"nih.gov).

We are working on an [IBEX JoVE methods collection](https://app.jove.com/methods-collections/3182) that will feature:
1. Imaging with a slide scanner by @nzangger.
2. Manual IBEX imaging with FFPE samples on slides using different instruments/imaging set ups. Happy to connect you to people who might be using a similar system.

Best of luck!
